### PR TITLE
SWAHILI.md file Translation.

### PR DESCRIPTION
**Pull request by the1Riddle**
hello @TenTraicion and @Syknapse
 
As it was mentioned in the issue #3017
i made the needed improvements on the `SWAHILI.md` file working on the format which was a problem in it.
some sections are also made to align with the recently updated readme file on the main repo's master branch.

just to mention an example is the [quick access section](https://github.com/Syknapse/Contribute-To-This-Project#quick-access-index)

hope it meets your satisfaction, and will be merged soon, if there is any part that would need to be edited, 
i would be ready to do so. 

address #3017 , thanks.

### CODE REVIEW

**Comment by the1Riddle** [comment]
address #2826

File: translations/README/SWAHILI.md [hunk 1]
```diff
@@ -132,24 +130,24 @@ _Muda uliokadiriwa: Chini ya dakika 30_.
...
 - Bofya _Clone_
 
-| ![Clone project](/readme-only/clone-project.PNG 'click on =your-github-username=/Changia-Kwa-Huu-Mradi') |
-| :---------------------------------------------- ----------------------------------------------- -------: |
+| ![Clone project](/readme-only/clone-project.PNG 'click on =your-github-username=/Contribute-To-This-Project') |
+| :----------------------------------------------------------------------------------------------------: |
 
-| :arrow_right_hook: Mradi uliogawanyika utakuwa na alama ya uma upande wa kushoto. Uma wako utakuwa na mtumiaji wako wa GitHub | ![uma wako](/readme-only/clone-your-fork.PNG 'uma wako utaonekana hivi, ukitumia jina lako la mtumiaji') |
-| :---------------------------------------------- ----------------------------------------------- --------------- | :---------------------------------------------- ----------------------------------------------- -: |
+| :arrow_right_hook: Mradi uliogawiwa utakuwa na ishara ya kugawanywa kushoto. Gawanya lako litakuwa na mtumiaji wako wa GitHub yenyewe. | ![your fork](/readme-only/clone-your-fork.PNG 'your fork will look like this, with your own user name') |
```
Comment (waynemorphic) [inline translations/README/SWAHILI.md:149]: Gawanyo lako litakuwa -> mgawanyo wako utakuwa

File: translations/README/SWAHILI.md [hunk 2]
```diff
@@ -1,50 +1,48 @@
...
+> Nembo imeundwa kwa :sparkling_heart: na [CandidDeer](https://github.com/CandidDeer)
 
 [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)][twit]
 
-[![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Jiunge%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Jiunge na seva yetu ya Discord !')
-[![PRs Karibu](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This -Mradi/)
-[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Changia-Kwa-Hii- Mradi/)
+[![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Join%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Join our Discord server!')
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This-Project/)
+[![Upendo wa Chanzo Wazi](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Contribute-To-This-Project/)
```
[the remark this answers is not in the document]
Comment (the1Riddle) [reply translations/README/SWAHILI.md:11]: You are right, its should be in English but still it was of no harm when i left it like that.
whats your thoughts , should i mend it...? @waynemorphic

[the remark this answers is not in the document]
Comment (TenTraicion) [reply translations/README/SWAHILI.md:11]: actually the links and references should be in English, otherwise they won't work.

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Recheck the files, fix if there is anything to fix and mention me after you finish rechecking. Thanks for the update.

File: translations/README/SWAHILI.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (the1Riddle) [reply translations/README/SWAHILI.md:11]: Awesome, i will make all the necessary changes suggested at once before i commit again.

**Comment by the1Riddle** [comment]
@TenTraicion 

********** **here are the changes made** ****************

1. changing the relevant links back to English
2. made updates from the suggestions made before
and also changed the word `uma` to `fork` as it would have had another meaning, different from the word forking.

File: translations/README/SWAHILI.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (the1Riddle) [reply translations/README/SWAHILI.md]: @waynemorphic what do you think about the current changes..?

**Review by waynemorphic — APPROVED** [review]
LGTM

**Comment by the-1Riddle** [comment]
> LGTM

Great .

**Review by TenTraicion — APPROVED** [review]
Seems Good

**Comment by TenTraicion** [comment]
@the1Riddle, Congratulations on your PR Merge, and Thanks for your Contribution.

Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Comment by the1Riddle** [comment]
> @the1Riddle, Congratulations on your PR Merge, and Thanks for your Contribution.
> 
> Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

Thanks for considering my contribution.
As i've already joined the discord community, am looking forward to also translate the `Maintainer's Guide` also to Swahili `soon`.

Best regards,

**Comment by TenTraicion** [comment]
sure, glad to hear that. if you need help or need to contact us, ask in Discord which is the fastest way. Happy Coding..!

**Review by kamranzafar4343 — COMMENTED** [review]
Looks great now.

**Review by olusiekwin — COMMENTED** [review]
LGTM

**Pull request merged**